app/module-types/node.js

```diff
@@ -35,9 +38,20 @@ exports.NodeModule = class {
 
   start () {
     if (this.script) {
+      const moduleLogPath = join(logPath, this.name)
```

Comment:
Here too. use `resolve`

Reply:
Ok? I just copied the behaviour of the script property, which used join